ast.go

```diff
@@ -104,3 +106,41 @@ type AST interface {
 	// Sections returns all Sections in the AST.
 	Sections() []Section
 }
+
+// ParseLine parses a single line.
+func ParseLine(s string) (l Line, err error) {
+	i, r := 0, []rune(s)
+	accept := func(f func(r rune) bool) (s String) {
+		if i < len(r) && f(r[i]) {
+			s.Pos = i
+			s.Text += string(r[i])
+			i++
+			for i < len(r) && f(r[i]) {
+				s.Text += string(r[i])
+				i++
+			}
+		}
+		return
+	}
+	l.Prefix = accept(func(r rune) bool {
+		return r == ' '
+	})
+	l.Keyword = accept(func(r rune) bool {
+		return r >= 'a' && r <= 'z' || r >= 'A' && r <= 'Z' || r >= '0' && r <= '9'
+	})
+	// TODO Parse l.Assignment
+	// TODO Parse l.Arguments
+	// TODO Parse l.Postfix
+	if i < len(r) && r[i] == '#' {
+		l.Comment = accept(func(r rune) bool {
+			return true
```

Comment:
Will not `l.Comment` contain `\n` rune too?

Reply:
It seems so. But current tests do not cover this and current code does not parse the NewLine (line 139). Next iteration that handles that will necessarily handle this.